pandas/core/nanops.py

```diff
@@ -683,9 +687,9 @@ def _get_counts(mask, axis, dtype=float):
         return np.array(count, dtype=dtype)
 
 
-def _maybe_null_out(result, axis, mask):
+def _maybe_null_out(result, axis, mask, min_count=1):
     if axis is not None and getattr(result, 'ndim', False):
-        null_mask = (mask.shape[axis] - mask.sum(axis)) == 0
+        null_mask = (mask.shape[axis] - mask.sum(axis) - min_count) < 0
```

Comment:
Can you just substract here something if `null_mask` is later on used for indexing into `result` ?

Reply:
I'm not sure what you mean here. `null_mask` will be a boolean array of which values in the output should be null because they have too few valid values.